sentry-core/Cargo.toml

```diff
@@ -31,9 +31,12 @@ UNSTABLE_cadence = ["dep:cadence", "UNSTABLE_metrics"]
 
 [dependencies]
 cadence = { version = "0.29.0", optional = true }
+crc32fast = "1.4.0"
+itertools = "0.13.0"
```

Comment:
are we still using this?

Reply:
similar to https://github.com/getsentry/sentry-rust/pull/659, we should make both these dependencies optional.
Also instead of manually sorting the metric tags, how about you switch to a `BTreeMap` which is sorted by definition?